### DISCUSSION TITLE
Github support doesn't answer

### DISCUSSION THREAD

**Discussion by mathieuj2024** [community/community#123945] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello!

I had to create a new account to send this message because my other account has been blocked by Github. I've been waiting for over a month now, and Github Support hasn't responded to my ticket #2723475 (username: mathieujelsch)! I received this message: "You are marked as spam, and therefore cannot authorize a third-party application."

Github, please, refrain from sending me emails about this forum not being intended for such inquiries. I just need a response to my ticket now! I won't stop posting here until my account is unblocked!

I hope my message will assist others facing the same issue.

Thanks

**Comment by ghostinhershell** [comment]
Hi @mathieuj2024,

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
